internals/overlord/logstate/loki/loki.go

```diff
@@ -182,7 +188,9 @@ func errFromResponse(resp *http.Response) error {
 	// Read response body to get more context
 	body, err := io.ReadAll(io.LimitReader(resp.Body, 1024))
 	if err != nil {
-		return fmt.Errorf("HTTP %d error, but cannot read response: %v", resp.StatusCode, err)
+		logger.Debugf("HTTP %d error, but cannot read response: %v", resp.StatusCode, err)
 	}
-	return fmt.Errorf("HTTP %d error, response %q", resp.StatusCode, body)
+	logger.Debugf("HTTP %d error, response %q", resp.StatusCode, body)
```

Comment:
Should this `logger.Debugf` be in an `else` clause?

Reply:
d'oh